samples/nrf9160/aws_fota/README.rst

```diff
@@ -261,8 +256,8 @@ ERROR: mqtt_connect -45:
    Certificates must be formatted correctly, without extra whitespace.
 
 Content range is not defined:
-   If you host the firmware image on a different server than nRF Cloud, this error indicates that the Content-Range field is missing in the HTTP GET header.
-   To fix this problem, configure the host server to provide this field.
+   If you host the firmware image on a different server than in an S3 bucket, this error indicates that the Content-Range field is missing in the HTTP GET header.
+   To fix this problem, configure the host server to provide this field. Also it could be that you have not configured the S3 bucket correctly. Please check that your file is available from the browser withouth being logged in to your AWS account.
```

Comment:
Add comma after "Also, it could be..."

withouth --> without

Reply:
```suggestion
   To fix this problem, configure the host server to provide this field. Also, it could be that you have not configured the S3 bucket correctly. Please check that your file is available from the browser without being logged in to your AWS account.
```